site/homepage/content/release-notes.md

```diff
@@ -17,6 +17,10 @@ description = "Information about changes in recent Hono releases. Includes new f
 * In the Quarkus variants of the MongoDB device registry and the Hono auth component, the provided metrics did not
   contain the default set of tags, as used in the other Hono components (e.g. *host* or *component-name*). This has been
   fixed.
+* The HTTP protocol adapter and Device Registry now support a configuration property for explicitly setting the idle timeout.
+  The timeout is configured with the property `idleTimeout`. This determines if a connection will timeout and be closed
+  if no data is received or sent within the idle timeout period. The idle timeout is in seconds.
+  A zero value means no timeout is used.
```

Comment:
This needs to go into the 1.12.3 section ... you might want to fetch and rebase, I have already added a 1.12.3 section ...